css/style.css

```diff
@@ -1,9 +1,203 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  display: flex;
+  flex-direction: column;
+  padding-left: 5%;
+  padding-right: 5%;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  margin-bottom: 0%;
+  padding-bottom: 0%;
+}
+
+.icodiv {
+  width: 20%;
+  margin-left: 1%;
+  padding-left: 1%;
+  margin-bottom: 0%;
+}
+
+.icoimage {
+  width: 15%;
+  height: 15%;
+}
+
+.navdiv {
+  width: 50%;
+  margin-bottom: 0%;
+}
+
+.unorderlist {
+  display: flex;
+  flex-direction: row;
+  padding: 5%;
+  justify-content: space-around;
+  list-style-type: none;
+  font-size: 20px;
+}
+
+a {
+  color: rgb(128, 118, 118);
+  text-decoration: none;
+}
+
+.meetkar {
+  font-weight: 600;
+}
+
+a:hover {
+  color: red;
+}
+
+main {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 0%;
+  margin-bottom: 10%;
+}
+
+.maindivimage {
+  position: relative;
+  width: 120%;
+  text-align: center;
+  color: white;
+}
+
+.mainimag {
+  max-width: 100%;
+  height: 80%;
+  padding: 5px;
+}
+
+.textimage1 {
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 48px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textimage2 {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 24px;
+}
+
+.button {
+  position: absolute;
+  top: 70%;
+  background-color: rgb(248, 127, 14);
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 20px;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+
+.bottomimages {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  margin-top: 0%;
+  width: 100%;
+}
+
+.divtext {
+  margin: 2%;
+  padding: 5%;
+  text-align: center;
+  font-size: 44px;
+  color: black;
+  font-weight: 500;
+}
+
+.image1 {
+  width: 33.33%;
+}
+
+.bimage {
+  width: 30%;
+  align-items: center;
+  transform: translate(120%, 10%);
+}
+
+.underimagetextr {
```

Comment:
When naming your classes, you can use 'kebab-case' which is when everything is lower case but you join each word with a hyphen `-`
In this example it would be `under-image-text`